scripts/dev/build_release.py

```diff
@@ -232,6 +233,9 @@ def build_windows():
 
     artifacts = []
 
+    utils.print_title("Updating VersionInfo file")
+    gen_versioninfo
```

Comment:
You're not actually calling anything here, and everything is at the toplevel of your script - that means it'll actually all be run when imported above.

Instead, the script should have a `def main():` function doing the work (everything after the imports), and a block like this at the bottom:

```python
if __name__ == '__main__':
    main()
```

then the script can still be run directly, but you can also run `gen_versioninfo.main()` here.